tests/wpt/metadata/html/browsers/browsing-the-web/back-forward-cache/events.html.ini

```diff
@@ -1,10 +1,7 @@
 [events.html]
   expected: TIMEOUT
-  [SameOrigin]
-    expected: TIMEOUT
-
   [SameSite]
-    expected: NOTRUN
+    expected: TIMEOUT
```

Comment:
this TIMEOUT three times so I think it was intermittent PASS during the sync CI job.

```
  ▶ Unexpected subtest result in /html/browsers/browsing-the-web/back-forward-cache/events.html:
  │ TIMEOUT [expected PASS] b'SameOrigin'
  └   → Test timed out

  ▶ Unexpected subtest result in /html/browsers/browsing-the-web/back-forward-cache/events.html:
  └ NOTRUN [expected TIMEOUT] b'SameSite'
```